### DISCUSSION TITLE
[Question] Would you mind if I ported the samples to .NET Core?

### DISCUSSION THREAD

**Discussion by davidhenley** [canton7/Stylet#225] [Q&A]
Would anyone mind if I created a PR to port all the samples to .NET Core?

It would drastically simplify the project files.

**Comment by canton7** [comment]
Apologies, this slipped under my radar.

I've been mulling this over. The problem is that I want the samples to be runnable by anybody: the barrier to entry needs to be as low as possible. Everyone has .NET Framework installed, but plenty of people won't have the right version(s) of .NET Core installed.

It would be possible to move to SDK-style csprojs for the samples, and perhaps multi-target them, but that requires using MSBuild.SDK.Extras. I use this for building Stylet itself, but it's again something which is a bit confusing, and not immediately obvious, about the samples.

I'd definitely be open to creating a .NET Core sample to show how it's done, but I'm leaning towards keeping most of the samples targetting .NET Framework as the lowest common demoninator.

**Comment by canton7** [comment]
(I'm converting this to a discussion. I know discussions weren't around when you opened this!)
